pkg/lwip/contrib/sock/ip/lwip_sock_ip.c

```diff
@@ -79,7 +80,8 @@ static uint16_t _ip4_addr_to_netif(const ip4_addr_p_t *addr)
     assert(addr != NULL);
 
     if (!ip4_addr_isany(addr)) {
-        for (struct netif *netif = netif_list; netif != NULL; netif = netif->next) {
+        struct netif *netif;
+        NETIF_FOREACH(netif) {
```

Comment:
Here and everywhere: Please add a cppcheck suppression comment, like suggested by the Github annotation.

Reply:
I am not sure why it was accepted in the [ifconfig](https://github.com/RIOT-OS/RIOT/commit/eab317749f82764c6e978ab13a6819b5320a031c#diff-989568003e3c26ef6bf5592983b5243864649c98090cf416e47ace07030bcce3R88) code but not here

Reply:
Not sure either. But `cppcheck` can be flaky like that.